Source/Graphic/HCPrimitive.c

```diff
@@ -0,0 +1,9 @@
+//
+//  HCPrimitive.c
```

Comment:
Were these files intended to be part of the `Graphics/` section? It seems like `HCPrimitive` could be a good name for some of the things in `HCCore` if we wanted to pull them out.

I also ask because there isn't much in these files.

Reply:
Yep. It has had more added as well in the next PR.